src/server/command/worldedit/mask.zig

```diff
@@ -0,0 +1,29 @@
+const std = @import("std");
+
+const main = @import("main");
+const User = main.server.User;
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+const Mask = main.blueprint.Mask;
+
+pub const description = "Set edit mask. When used with no mask expression it will clear current mask.";
+pub const usage =
+	\\/mask <mask>
+	\\/mask
```

Comment:
Not something that needs to be solved here:
It is not quite obvious what `/mask` without arguments actually does. I think we need to add a convention for extra descriptions to the the help menu here.

Reply:
Well, ignoring the explanation in description, yes, this might be confusing.

Reply:
Well, no one will read a big description that explains every single subcommand.